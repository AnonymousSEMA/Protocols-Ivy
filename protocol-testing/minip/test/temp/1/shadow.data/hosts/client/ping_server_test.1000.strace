00:00:00.000000000 [tid 1000] shadow_init_memory_manager(...) = 0
000000000000000000 [tid 1000] gettimeofday(...) = 0
00:00:00.000000000 [tid 1000] openat(-100, "/sys/devices/system/cpu", O_NONBLOCK | O_DIRECTORY | O_CLOEXEC, Mode(0x0)) = 3
00:00:00.000000000 [tid 1000] fstat(...) = 0
00:00:00.000000000 [tid 1000] getdents64(...) = 752
00:00:00.000000000 [tid 1000] getdents64(...) = 0
00:00:00.000000000 [tid 1000] close(3) = 0
00:00:00.000000000 [tid 1000] sched_getaffinity(192, 8, 0x5555555f0070) = -3 (ESRCH)
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] openat(-100, "/sys/devices/system/cpu/online", O_CLOEXEC, Mode(0x0)) = 3
00:00:00.000000000 [tid 1000] read(3, 0x7fffffffc480, 8192) = 2
00:00:00.000000000 [tid 1000] close(3) = 0
00:00:00.000000000 [tid 1000] futex(...) = 0
00:00:00.000000000 [tid 1000] futex(...) = 0
00:00:00.000000000 [tid 1000] setrlimit(...) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 0, as_u64: 0, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 0 } }
000000000000000000 [tid 1000] time(...) = 946684800
00:00:00.000000000 [tid 1000] rt_sigaction(...) = 0
00:00:00.000000000 [tid 1000] rt_sigaction(...) = 0
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "http_request_file.txt", 21) = 21
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "PROOTPATH 5b ", 13) = 13
00:00:00.000000000 [tid 1000] write(2, "/PFV", 4) = 4
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "/PFV/QUIC-Ivy-Attacker/doc/examples/quic"..., 67) = 67
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] open("/PFV/QUIC-Ivy-Attacker/doc/examples/quic"..., O_RDWR, Mode(0x0)) = -2 (ENOENT)
00:00:00.000000000 [tid 1000] write(2, "cannot open file: No such file or direct"..., 44) = 44
00:00:00.000000000 [tid 1000] write(2, "-1", 2) = 2
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] brk(0x55555562c000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993116160, as_u64: 93824993116160, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993116160 } }
00:00:00.000000000 [tid 1000] brk(0x55555564d000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993251328, as_u64: 93824993251328, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993251328 } }
00:00:00.000000000 [tid 1000] brk(0x55555566f000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993390592, as_u64: 93824993390592, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993390592 } }
00:00:00.000000000 [tid 1000] brk(0x555555690000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993525760, as_u64: 93824993525760, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993525760 } }
00:00:00.000000000 [tid 1000] brk(0x5555556b1000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993660928, as_u64: 93824993660928, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993660928 } }
00:00:00.000000000 [tid 1000] brk(0x5555556d2000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993796096, as_u64: 93824993796096, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993796096 } }
00:00:00.000000000 [tid 1000] brk(0x5555556f3000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993931264, as_u64: 93824993931264, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993931264 } }
00:00:00.000000000 [tid 1000] brk(0x555555714000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994066432, as_u64: 93824994066432, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994066432 } }
00:00:00.000000000 [tid 1000] brk(0x555555735000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994201600, as_u64: 93824994201600, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994201600 } }
00:00:00.000000000 [tid 1000] brk(0x555555756000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994336768, as_u64: 93824994336768, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994336768 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555786000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994533376, as_u64: 93824994533376, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994533376 } }
00:00:00.000000000 [tid 1000] brk(0x5555557a8000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994672640, as_u64: 93824994672640, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994672640 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x5555557d3000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994848768, as_u64: 93824994848768, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994848768 } }
00:00:00.000000000 [tid 1000] brk(0x5555557f4000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994983936, as_u64: 93824994983936, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994983936 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555822000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995172352, as_u64: 93824995172352, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995172352 } }
00:00:00.000000000 [tid 1000] brk(0x555555843000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995307520, as_u64: 93824995307520, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995307520 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555864000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995442688, as_u64: 93824995442688, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995442688 } }
00:00:00.000000000 [tid 1000] brk(0x555555885000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995577856, as_u64: 93824995577856, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995577856 } }
00:00:00.000000000 [tid 1000] brk(0x5555558a6000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995713024, as_u64: 93824995713024, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995713024 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x5555558c7000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995848192, as_u64: 93824995848192, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995848192 } }
00:00:00.000000000 [tid 1000] brk(0x5555558fb000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996061184, as_u64: 93824996061184, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996061184 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x55555592f000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996274176, as_u64: 93824996274176, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996274176 } }
00:00:00.000000000 [tid 1000] brk(0x555555950000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996409344, as_u64: 93824996409344, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996409344 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] write(2, "COUCOU", 6) = 6
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] lseek(...) = -9 (EBADF)
00:00:00.000000000 [tid 1000] write(2, "cannot seek to beginning of file: Bad fi"..., 54) = 54
00:00:00.000000000 [tid 1000] read(-1, 0x55555592be80, 1024) = -9 (EBADF)
00:00:00.000000000 [tid 1000] write(2, "recvfrom failed\n", 16) = 16
00:00:00.000000000 [tid 1000] exit_group(...) = <native>
