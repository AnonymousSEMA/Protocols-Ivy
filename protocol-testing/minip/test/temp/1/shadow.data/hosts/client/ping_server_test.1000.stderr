"http_request_file.txt"
PROOTPATH 5b /PFV
"/PFV/QUIC-Ivy-Attacker/doc/examples/quic/ping/http_request_file.txt"
cannot open file: No such file or directory
-1
COUCOU
cannot seek to beginning of file: Bad file descriptor
recvfrom failed
